spec.html

```diff
@@ -16564,7 +16572,7 @@
               for (const key of Reflect.ownKeys(obj)) {
                 if (typeof key === "symbol") continue;
                 const desc = Reflect.getOwnPropertyDescriptor(obj, key);
-                if (desc && !visited.has(key)) {
+                if (desc) {
```

Comment:
We still need to track when the same property key is also present in the prototype chain.